src/js/jsx/sections/pages/LayerFace.jsx

```diff
@@ -40,16 +40,48 @@ define(function (require, exports, module) {
         system = require("js/util/system"),
         strings = require("i18n!nls/strings");
 
+    /*
+     * Function for checking whether React component should update
+     * Passed to Droppable composed component in order to save on extraneous renders
+     *
+     * @param {Object} nextProps - Next set of properties for this component
+     * @return {boolean}
+     */
+    var shouldComponentUpdate = function (nextProps) {
```

Comment:
Actually...I can probably kill this call all together (since we have already used the same thing on Droppable)